packages/tools/examples/stackLabelmapSegmentation/index.ts

```diff
@@ -112,11 +107,11 @@ const optionsValues = [
   ...brushValues,
   RectangleScissorsTool.toolName,
   CircleScissorsTool.toolName,
-  SphereScissorsTool.toolName,
   PaintFillTool.toolName,
 ];
 
 let viewport;
+const viewportId2 = 'STACK_VIEWPORT_2';
```

Comment:
Could you create an import that has viewportId1='viewportId1' to viewportId4 or whatever you use, then just import them?   Seems like using a consistent set would be useful.  Can be done later, just that it is annoying to copy an example and find the ids no longer match up.

Reply:
Yes, later sounds good